spec-main/api-web-contents-spec.ts

```diff
@@ -1541,6 +1541,39 @@ describe('webContents module', () => {
     });
   });
 
+  describe('getBackgroundThrottling()', () => {
+    afterEach(closeAllWindows);
+    it('works via getter', () => {
```

Comment:
Maybe add an empty line after `afterEach`?

Reply:
it would be nice, but I don't want to trigger the builds again